src/components/Accordion/Accordion.tsx

```diff
@@ -49,13 +60,20 @@ const getIcon = (isExpanded = false, index: number, id: string) => {
 const getElementsFromData = (
   data: AccordionDataProps[] = [],
   id: string,
-  panelMaxHeight: string
+  panelMaxHeight: string,
+  isDarkMode: boolean
 ) => {
-  const colorMap = {
-    default: "ui.white",
-    warning: "ui.status.primary",
-    error: "ui.status.secondary",
-  };
+  const colorMap = isDarkMode
+    ? {
+        default: "ui.white",
+        warning: "ui.status.primary",
```

Comment:
Should this be `dark.ui.status.primary`?

Reply:
@bigfishdesign13 the color in the FIgma design is `ui.status.primary`, however it makes sense to be `dark.ui.status.primary`.

Reply:
@jackiequach What you have in the code is correct.  The `dark` color variants were not used because the colors were used on such a small piece.  With that treatment, the colors needed to be brighter and more intense than the `dark` variants.  Do not change what you have.